# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../elvis-tiny/compile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The usual type conflict cought by gcc >= 3.4.x.

  - Rene Rebe <rene@exactcode.de>

diff -ur elvis-tiny-1.4.orig/config.h elvis-tiny-1.4.orig-patched/config.h
--- elvis-tiny-1.4.orig/config.h	1991-08-04 23:20:32.000000000 +0200
+++ elvis-tiny-1.4.orig-patched/config.h	2005-02-02 19:22:42.021908344 +0100
@@ -178,13 +178,6 @@
 # define uchar		unsigned char
 #endif
 
-/* Some compilers prefer to have malloc declared as returning a (void *) */
-#if BSD
-extern void *malloc();
-#else
-extern char *malloc();
-#endif
-
 /* Most compilers could benefit from using the "register" storage class */
 #if 1
 # define REG	register
